operator/validators/validators_map.go

```diff
@@ -139,6 +141,10 @@ func (vm *ValidatorsMap) GetCommittee(pubKey spectypes.CommitteeID) (*validator.
 
 	v, ok := vm.committees[pubKey]
 
+	if !ok || v.Stopped() {
+		return nil, false
+	}
```

Comment:
It doesn't seem like "stopped" committee can be present in `vm.committees` map because we immediately remove stopped committees from `UpdateCommitteeAtomic`, so perhaps no need to add this check here ?